examples/configure_hydra/workdir/conf/config.yaml

```diff
@@ -5,3 +5,5 @@ hydra:
   sweep:
```

Comment:
now that the bulk of the docs are no longer under configure_hydra, move the example app accordingly.

Reply:
Not sure what you mean here 🤔 the docs are still under configure_hydra.

Reply:
got confused. looks okay.